src/commands/dev.ts

```diff
@@ -54,23 +54,21 @@ import {
   isYarn,
   openInBrowser,
   resolveDependencyManifest,
-  findMatchingMountScript,
   updateLockfileHash,
 } from '../util';
 import {
   FileBuilder,
+  generateEnvModule,
```

Comment:
MRW code is alphabetized

![](https://media.giphy.com/media/7aPevwPAgsBkQ/giphy.gif)